src/api/app/models/owner_search/base.rb

```diff
@@ -0,0 +1,87 @@
+require 'api_exception'
+
+module OwnerSearch
+  class Base
+    class AttributeNotSetError < APIException
+      setup 'attribute_not_set', 400
+    end
+
+    protected
+
+    def initialize(params = {})
+      self.params = params
+      self.attribute = AttribType.find_by_name!(params[:attribute] || 'OBS:OwnerRootProject')
+
+      self.limit = (params[:limit] || 1).to_i
+    end
+
+    attr_accessor :params, :attribute, :limit
+
+    def object_projects(_obj)
```

Comment:
why passing ```_obj``` here if its not being used?

Reply:
now that's a bummer

Reply:
renamed the function now - it longer takes an obj